examples/example_2.py

```diff
@@ -36,7 +36,7 @@ def example_opt_numfreq(orca_code, submit=True):
                 },
             },
             'input_kewords': ['RKS', 'BP', 'def2-SVP', 'RI', 'def2/J'],
-            'extra_input_keywords': ['Grid2', 'NoFinalGrid', 'NumFreq', 'OPT'],
+            'extra_input_keywords': ['DEFGRID1', 'NumFreq', 'OPT'],
```

Comment:
Do we need to specify grid here? If we just used default we could support both ORCA versions.

Reply:
Good point. I agree.